app/src/main/java/org/eyeseetea/malariacare/data/remote/SdkController.java

```diff
@@ -104,13 +104,13 @@ public static void postFinish() {
 
     public static boolean finishPullJob() {
```

Comment:
in this branch only create data from csv's in new architecture way, in next branch I will refactor this